lib/pbench/agent/tool_meister.py

```diff
@@ -1285,6 +1285,42 @@ def stop_tools(self, data):
             self._send_client_status("success")
         return failures
 
+    def _create_tar(
+        self,
```

Comment:
If it make sense, should this be made a `utils` functionality and should this reside in `lib/pbench/common/utils.py` just like how we do it with `md5sum`?

Reply:
Right now the server has no interest in creating a tarball, although we might eventually come up with a use case; so `agent/utils.py` might make more sense. Then again, the only other place I think we do this, where we're preparing to push data up to a server, is currently in `bash`.